crates/noirc_evaluator/src/ssa/opt/flatten_cfg.rs

```diff
@@ -1040,54 +1040,54 @@ mod test {
             builder.insert_store(r1, value);
         };
 
-        let println = builder.import_intrinsic_id(Intrinsic::Println);
+        let sort = builder.import_intrinsic_id(Intrinsic::Sort);
```

Comment:
Instead of Sort, can we create a new function id (via `test_new`) and call that function?

Reply:
Line 1003 also needs to be updated since it mentions println